index.js

```diff
@@ -1,7 +1,19 @@
 const farmAnimals = 'cow horse sheep pig chicken';
 
+const [moo, neigh, baa, oink, cluck] = farmAnimals.split(" ")
```

Comment:
`farmAnimals` doesn't need to be `split`. 
```suggestion
const [moo, neigh, baa, oink, cluck] = farmAnimals
```